parsec/backend/postgresql/user_queries/get.py

```diff
@@ -304,9 +303,9 @@ async def query_get_user_with_devices_and_trustchain(
             d.redacted_device_certificate if redacted else d.device_certificate
             for d in user_devices
         ],
-        trustchain_device_certificates=trustchain.devices,
-        trustchain_user_certificates=trustchain.users,
-        trustchain_revoked_user_certificates=trustchain.revoked_users,
+        trustchain_device_certificates=list(trustchain.devices),
```

Comment:
I think it should be `GetUserAndDevicesResult`'s definition that should be corrected instead to use tuples instead of lists, we love immutability around here ;-)

Reply:
9ddb38c5be108a3fc89d6ff4ab2404d92401c0ed